source/server/spongineer/incompatible.rst

```diff
@@ -84,11 +84,13 @@ Just Enough IDs (JEID)
 - Problem: Crash on startup (mixin conflict).
 - Solution: Update to version 1.0.3-54 or newer.
 
-LagGoggles
-~~~~~~~~~~
-- Versions: Older than 1.12.2-5.3-113
+LagGoggles/TickCentral
```

Comment:
Isn't `Tickcentral` the coremod? We can just list that as a dependency, same as the others

Reply:
TickCentral is, but older versions of LagGoggles before TickCentral came about also have problems with SpongeForge. I didn't see much of a benefit in making 2 rows that mirror each other in this case, when they are both from the same author.

Reply:
The only reason I list them is so users know what else to remove. It'll do for now.